src/main/java/org/cactoos/scalar/MultiplicationOf.java

```diff
@@ -81,7 +81,7 @@ public MultiplicationOf(final Iterable<? extends Number> src) {
             return new Folded<>(
                 BigDecimal.ONE,
                 (mtn, value) -> mtn.multiply(value, MathContext.DECIMAL128),
-                    new Mapped<>(
+                    new Mapped<BigDecimal>(
```

Comment:
@victornoel This issue with type inference here is caused by either older JDK, or IDE version.

Reply:
@andreoss yes, it's Eclipse, but it's not incorrect to explicit the type here, so let's keep it, at least people using Eclipse will be able to contribute without bothering with those errors